src/routes/safe/components/Balances/SendModal/screens/AddressBookInput/style.ts

```diff
@@ -4,18 +4,18 @@ export const useTextFieldLabelStyle = makeStyles(
   createStyles({
     root: {
       overflow: 'hidden',
-      borderRadius: 4,
-      fontSize: '15px',
-      width: '500px',
     },
   }),
 )
 
 export const useTextFieldInputStyle = makeStyles(
   createStyles({
     root: {
-      fontSize: '14px',
-      width: '420px',
+      fontSize: '16px',
+      padding: '0 !important',
```

Comment:
Is this address book-specific? You could move these into the global overrides and now have to set them as `!important`

Reply:
We can move some of it into the global overrides but will still need to use !important I am afraid since the default mui selectors are more specific